src/utils/helpers.ts

```diff
@@ -160,4 +160,21 @@ export async function getCommitId(): Promise<string> {
 
 export function statusCodeToTitle(code: number): AppDeploymentStatus {
   return ["NEW", "TSS_GROUP_SELECTED", "DEPLOYED", "PENDING", "EXPIRED"][code] as AppDeploymentStatus;
+}
+
+export function numToUint8Array(num) {
```

Comment:
Can't we convert timestamps to string and save as strings?

Something like this:

```javascript
import { fromString as uint8ArrayFromString, toString as uint8ArrayToString} from 'uint8arrays/from-string'

timestamp = parseInt(uint8ArrayToString(....))
unit8ArrTimestamp = uint8ArrayFromString(time.toString())
```

We do not need these 2 funcs anymore.